src/modules/partners/partner_service.ts

```diff
@@ -5,15 +5,20 @@ export namespace Partner {
   export const findAll = async (requestQuery: Entity.RequestQuery): Promise<Entity.ResponseFindAll> => {
     const items: any = await Repository.findAll({
       name: requestQuery.name,
-      perPage: requestQuery.perPage,
-      currentPage: requestQuery.currentPage
+      per_page: requestQuery.per_page,
+      current_page: requestQuery.current_page
     })
 
     const result: Entity.ResponseFindAll = {
       data: items.data,
       meta: {
-        lastUpdate: items.data.length > 0 ? items.data[0].created_at : null,
-        ...items.pagination
+        last_update: items.data.length > 0 ? items.data[0].created_at : null,
```

Comment:
I think `items.data.length` is simply enough, since you're only returning a boolean

Reply:
thanks @khihadysucahyo :smile: